index.html

```diff
@@ -29,7 +29,7 @@
           </tr>
           <tr>
             <td>Remaining Calories</td>
-            <td class="remaining-calories"></td>
+            <td></td>
```

Comment:
I thought that adding a class to it might not be the right way to do things but that was my first thought. I think it would also allow me to get rid of an extra method so I'll go ahead and do it that way.